tracer/src/Datadog.Trace/Ci/CIEnvironmentValues.cs

```diff
@@ -600,7 +600,12 @@ private void SetupAppveyorEnvironment()
                 Branch = EnvironmentHelpers.GetEnvironmentVariable("APPVEYOR_REPO_BRANCH");
             }
 
-            Message = EnvironmentHelpers.GetEnvironmentVariable("APPVEYOR_REPO_COMMIT_MESSAGE_EXTENDED");
+            Message = EnvironmentHelpers.GetEnvironmentVariable("APPVEYOR_REPO_COMMIT_MESSAGE");
+            string extendedMessage = EnvironmentHelpers.GetEnvironmentVariable("APPVEYOR_REPO_COMMIT_MESSAGE_EXTENDED");
+            if (!string.IsNullOrWhiteSpace(extendedMessage))
+            {
+                Message = Message + "\n" + extendedMessage;
+            }
             AuthorName = EnvironmentHelpers.GetEnvironmentVariable("APPVEYOR_REPO_COMMIT_AUTHOR");
```

Comment:
Causing build failures due to linting:
```suggestion
            }

            AuthorName = EnvironmentHelpers.GetEnvironmentVariable("APPVEYOR_REPO_COMMIT_AUTHOR");
```